com.longbridgeapp.LongbridgePro.metainfo.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.longbridgeapp.LongbridgePro</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary=https://support.longbridgeapp.com/topics/us-trade/user-agreement</project_license>
```

Comment:
The license is in Chinese (?) I think. Is redistribution allowed? Did you contact them?